### PR TITLE
Add ability to toggle wireframes for markers

### PR DESCRIPTION
**Pull request by snosenzo**
**User-Facing Changes**

 - Add ability to toggle wireframes for markers

**Description**
All marker and scene entity topics now have the ability to toggle wireframes of all renderable types that have them. Initially tried giving people the ability to recolor to control opacity, but I think people generally will want it to be on or off~~but the `LineBasicMaterial` doesn't support opacity when rendering.~~

FG-2627

### CODE REVIEW

File: packages/studio-base/src/panels/ThreeDeeRender/renderables/primitives/RenderablePrimitive.ts [hunk 1]
```diff
@@ -15,7 +16,28 @@ export type EntityRenderableUserData = BaseUserData & {
...
+    renderer: Renderer,
+    userData: EntityRenderableUserData = {
+      receiveTime: -1n,
+      messageTime: -1n,
+      frameId: "",
+      pose: emptyPose(),
+      settings: PRIMITIVE_DEFAULT_SETTINGS,
+      settingsPath: [],
+      entity: undefined,
+    },
```
Comment (jtbandes) [inline packages/studio-base/src/panels/ThreeDeeRender/renderables/primitives/RenderablePrimitive.ts:37]: I've been wondering if we could generally do away with these made-up default values on user data...

**Review by jtbandes — APPROVED** [review]
Looks good but please add stories :)

**Comment by jtbandes** [comment]
> `LineBasicMaterial` doesn't support opacity when rendering.


Thoughts? https://stackoverflow.com/questions/63647873/how-to-set-opacity-in-drawing-lines-use-three-js-use-linebasicmaterial

**Comment by snosenzo** [comment]
> > `LineBasicMaterial` doesn't support opacity when rendering.
> 
> Thoughts? https://stackoverflow.com/questions/63647873/how-to-set-opacity-in-drawing-lines-use-three-js-use-linebasicmaterial

Really not sure how I got to that conclusion at the time, my mistake. I do think having a toggle makes sense given the original ask. My initial thinking was that they can just control the opacity via color, but in most cases I imagine a user would primarily want to just toggle the outline on/off. I have another branch with code for the other way if we decide to do it though.

**Comment by jhurliman** [comment]
Too bad this wasn't implemented as an RGBA color where opacity could be used to hide the lines, but it would also allow coloring the wireframe. Maybe a future update @snosenzo?

**Pull request merged**